EDEngineer/Resources/Data/blueprints.json

```diff
@@ -27282,6 +27282,701 @@
+        "Name": "Tungsten",
+        "Size": 4
+      },
+      {
+        "Name": "Electrochemical Arrays",
+        "Size": 6
+      },
+	  {
+	   "Name": "Mechanical Components",
+	   "Size": 4
+	  },
+      {
+        "Name": "Boron",
+        "Size": 4
+      }
+    ],
+    "Grade": 2
+  },
+  {
+    "Type": "Flechette Launcher Munitions",
+	"Name": "+10% Damage Boost",
+	"Engineers": [
+	  "@Synthesis"
+	],
+	"Ingredients": [
+      {
+        "Name": "Tungsten",
+        "Size": 6
+      },
+      {
+        "Name": "Electrochemical Arrays",
+        "Size": 9
+      },
+	  {
+	   "Name": "Mechanical Components",
+	   "Size": 5
+	  },
+      {
+        "Name": "Boron",
+        "Size": 6
+      }
+    ],
+    "Grade": 3
+  },
+  {
+    "Type": "Guardian Shard Cannon Munitions",
+	"Name": "None",
+	"Engineers": [
+	  "@Synthesis"
+	],
+	"Ingredients": [
+      {
+        "Name": "Carbon",
+        "Size": 3
+      },
+      {
+        "Name": "Vanadium",
+        "Size": 2
+      },
+      {
+        "Name": "Crystal Shards",
+        "Size": 3
+      },
+	  {
+	   "Name": "Guardian Power Cell",
+	   "Size": 3
+	  },
+      {
+        "Name": "Guardian Sentinel Wreckage Components",
+        "Size": 5
+      }
+    ],
+    "Grade": 1
+  },
+  {
+    "Type": "Guardian Shard Cannon Munitions",
+	"Name": "+15% Damage Boost",
+	"Engineers": [
+	  "@Synthesis"
+	],
+	"Ingredients": [
+      {
+        "Name": "Crystal Shards",
+        "Size": 4
+      },
+      {
+        "Name": "Guardian Power Cell",
+        "Size": 2
+      },
+      {
+        "Name": "Guardian Sentinel Weapon Parts",
+        "Size": 2
+      }
+    ],
+    "Grade": 2
+  },
+  {
+    "Type": "Guardian Shard Cannon Munitions",
+	"Name": "+30% Damage Boost",
+	"Engineers": [
+	  "@Synthesis"
+	],
+	"Ingredients": [
+      {
+        "Name": "Carbon",
+        "Size": 8
+      },
+      {
+        "Name": "Selenium",
+        "Size": 3
+      },
+      {
+        "Name": "Vanadium",
+        "Size": 4
+      },
+      {
+        "Name": "Crystal Shards",
+        "Size": 8
+      }
+    ],
+    "Grade": 3
+  },
+  {
+    "Type": "AX Explosive Munitions",
+	"Name": "None",
+	"Engineers": [
+	  "@Synthesis"
+	],
+	"Ingredients": [
+      {
+        "Name": "Carbon",
+        "Size": 4
+      },
+      {
+        "Name": "Iron",
+        "Size": 3
+      },
+      {
+        "Name": "Nickel",
+        "Size": 3
+      },
+      {
+        "Name": "Propulsion Elements",
+        "Size": 3
+      }
+    ],
+    "Grade": 1
+  },
+  {
+    "Type": "AX Explosive Munitions",
+	"Name": "+5% Damage",
+	"Engineers": [
+	  "@Synthesis"
+	],
+	"Ingredients": [
+      {
+        "Name": "Sulphur",
+        "Size": 6
+      },
+      {
+        "Name": "Phosphorus",
+        "Size": 6
+      },
+      {
+        "Name": "Mercury",
+        "Size": 2
+      },
+      {
+        "Name": "Propulsion Elements",
+        "Size": 4
+      },
+	  {
+	    "Name": "ThargoidOrganic Circuitry",

```

Comment:
think there's a space missing here

Reply:
Blast it, will fix after work

Reply:
all right no worries I'll fix that it'll be quick